README.md

```diff
@@ -26,6 +26,26 @@ again, and was frustrutated when I couldn't find a similar plugin for vim, so
 after reading Steve Losh's 'Learn Vimscript the Hard Way' I decided to write one
 for myself.
 
+### Installation
+
+Using vim >=8.0, the builtin *packages* system is recommended:
```

Comment:
Actually vim's package mechanism works quite similar to pathogen, without the overhead. But I can easily add a few lines to that to also include pathogen, vundle, vim-plug and NeoBundle.